challenge_one/README.md

```diff
@@ -0,0 +1,23 @@
+# Nethermind Bot Deploy and Update Monitor
+
+## Description
+
+This agent detects when Nethermind deploys or updates a Forta Bot
+
+## Supported Chains
+
+- Polygon
+
+## Alerts
+
+- FORTA-BOT-UPDATE
+  - Fired when a transaction comes from Nethermind's wallet address that contains the agentUpdate function call
+
+- FORTA-BOT-CREATE
+  - Fired when a transcation comes from Netherminds wallet address that contains the createAgent function call
+
+## Test Data
+
+The agent behaviour can be verified with the following transactions:
+
+- 0x72482c5bddd5ee333e177caf8cd4056a8eee83b74cdcc4218b8fa707854e66f4 (bot creation with id: 64235728982376409709439139978057092858568305508934039865220382119682812608133)
```

Comment:
```suggestion
- [0x72482c5bddd5ee333e177caf8cd4056a8eee83b74cdcc4218b8fa707854e66f4](https://polygonscan.com/tx/0x72482c5bddd5ee333e177caf8cd4056a8eee83b74cdcc4218b8fa707854e66f4) (bot creation with id: 64235728982376409709439139978057092858568305508934039865220382119682812608133)
```
using `[tx-hash](block-explorer-url-of-the-tx)` converts the example hash to a link